docs/pages/markdown guide.md

```diff
@@ -0,0 +1,38 @@
+# Markdown Guide for documentation
+
+1. Number symbol followed by a space is used for marking headings in bold. Additonal number sympols allow for subheadings (# then ## then ###, etc.)
+
+1. The number 1 then a period and a space (1. ) is used to create a numbered list. Each section of the list needs the 1. and Visual Studio will automatically number each selection
+
+1. App level markups are done with an accent a grave ` on either side of the word(s) to be highlighred
+
+`Apps`
```

Comment:
There might be some better ways to show this off. Take a look at the [myst docs](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#show-backticks-inside-raw-markdown-blocks) on showing backticks